.github/workflows/build-and-test-windows.yml

```
@@ -112,20 +112,19 @@ jobs:
         github.event.label.name == 'run-windows-build' ||
         (github.event.review.state == 'APPROVED' && contains(github.event.pull_request.labels.*.name, 'requires-windows-build'))
     name: Lint Assets
-    uses: Lombiq/NodeJs-Extensions/.github/workflows/lint.yml@dev
+    uses: Lombiq/GitHub-Actions/.github/workflows/asset-lint.yml@issue/OSOE-1185
     # Be sure to keep the configs in sync with the build-and-test workflow.
     with:
-      scripts: '
-        {
-          "src/Libraries/Lombiq.EInvoiceValidator/Lombiq.EInvoiceValidator": { "scripts": { "source": "." } },
-          "src/Modules/Lombiq.ContentEditors/Lombiq.ContentEditors": { "scripts": { "source": "wwwroot/js" } },
-          "src/Modules/Lombiq.ContentEditors/Lombiq.ContentEditors.Samples": { "scripts": { "source": "wwwroot/js" } },
-          "src/Modules/Lombiq.DataTables/Lombiq.DataTables": { "scripts": { "source": "wwwroot/js" } },
-          "src/Modules/Lombiq.HelpfulExtensions/Lombiq.HelpfulExtensions": { "scripts": { "source": "wwwroot/js" } },
-          "src/Modules/Lombiq.UIKit/Lombiq.UIKit": { "scripts": { "source": "wwwroot/js" } },
-          "src/Modules/Lombiq.Walkthroughs/Lombiq.Walkthroughs": { "scripts": { "source": "wwwroot/js" } },
-        }'
-      styles-css: >-
+      scripts: >
+          src/Libraries/Lombiq.EInvoiceValidator/Lombiq.EInvoiceValidator : .,
```

Typo? Same in the other workflow.

```suggestion
          src/Libraries/Lombiq.EInvoiceValidator/Lombiq.EInvoiceValidator,
```

---

No, intentional. Lombiq.EInvoiceValidator's scrips are in the project root.
You can see the same in the NE config [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/1156/files/f0db5506e023e95fd12186614d8fab3ea420f1ae#diff-5ca69371a0b431061a0b1b145a164ac5797c6a8bd745fba6061e3afe213318fdL120) (`{ "source": "." }`)